### ISSUE TITLE
[BUG] URL to faulty module's page is run-in with following sentence, merging the two

### ISSUE THREAD

**Issue by mkahvi** [ruipin/fvtt-lib-wrapper#73]
**Confirm you have read the above**
The bug is in libWrapper's own error about it not being libWrapper error.

**Describe the bug**
"Report this issue here:" text is run in with previous URL, causing the "Report" part to be part of the URL.

Example of the bit in question:
```
Find information about this module here: https://gitlab.com/koboldworks/pf1/sheetReport this issue here:
```
Notice how `Report` is run-in with the URL?

**To Reproduce**
1. Cause an error that libWrapper reports.

**Expected behavior**
The link to not include "Report" from the following sentence.

**Screenshots**
![Screenshot 2022-11-14 170158](https://user-images.githubusercontent.com/8120158/201693754-405f918a-7864-49c6-8797-7fd093e77876.png)

**Technical Details (please complete the following information):**
 - LibWrapper Version: 1.12.10.0
 - FoundryVTT Version: 10.290

**Comment by ruipin** [comment]
Thank you so much for this detailed bug report! I don't have time right now to look into it, but I'll try to find some time this week.

**Comment by ruipin** [comment]
What browser are you using?
Also what version of the pf1 sheet module are you using? Any other requirements to see this exact message?

I am unable to reproduce, unfortunately. All such errors I can generate myself have the proper line-break at that position.
Code inspection does not show any obvious code path where this should be possible, so I am probably missing something.

**Comment by mkahvi** [comment]
It appeared while I was developing the module more and I cvaused bugs.

PF1 0.82.2, Foundry 10.290 with Electron client, ...

I don't think there's anything unusual about the URL field either to cause it: <https://gitlab.com/koboldworks/pf1/sheet/-/blob/master/release/module.json#L37>

**Comment by mkahvi** [comment]
That I do not remember. And I don't remember what the error itself was. I can try making something break, I guess.

**Comment by ruipin** [comment]
Nevermind, I think I found the problem.

**Comment by ruipin** [comment]
Fixed in [v1.12.11.0](https://github.com/ruipin/fvtt-lib-wrapper/releases/tag/v1.12.11.0). Thanks for your help.

Turns out I had broken this in [v1.12.9.0](https://github.com/ruipin/fvtt-lib-wrapper/releases/tag/v1.12.9.0).

Closing this, let me know if you encounter this again.
